ui/src/components/Tasks/Widgets/NumericWithUnitWidget/NumericWithUnitWidget.jsx

```diff
@@ -0,0 +1,42 @@
+import { Form } from 'react-bootstrap';
+
+import styles from './NumericWithUnitWidget.module.css';
+/**
+ *
+ * @param {import('@rjsf/utils').WidgetProps} props
+ * @returns
+ */
+export default function NumericWithUnitWidget({
```

Comment:
This is used for all numeric inputs, not just those with units, so I would rename it `NumericWidget`